geniza/common/management/commands/visual_tests.py

```diff
@@ -38,6 +38,29 @@ def take_snapshots(self, browser, dark_mode=False):
 
         # content page
         browser.get("http://localhost:8000/en/content/")
+        # disable srcset and swap out content page images for ones from /static that Percy can see
```

Comment:
ah — is this your workaround for the uploaded user-content images?

let's flag it a little more loudly and clearly here in the comments with a NOTE and explain why we're doing it